ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="left_arm-eb2-j0_1-inertial" type="embObjIMU">
+
+        <xi:include href="../../general.xml" />
+
+        <xi:include href="../../hardware/electronics/left_arm-eb2-j0_1-eln.xml" />
+
+        <group name="SERVICE">
+
+            <param name="type"> eomn_serv_AS_inertials3 </param>
+
+            <group name="PROPERTIES">
+
+                <group name="CANBOARDS">
+                    <param name="type">             eobrd_strain2           </param>
+
+                    <group name="PROTOCOL">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                    </group>
+
+                    <group name="FIRMWARE">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                        <param name="build">            0                   </param>
+                    </group>
```

Comment:
These two versions set to 0 are fine @marcoaccame @sgiraz @davidetome ?

Reply:
Hi @Nicogene,

Actually, `strain2` and `strain2c` have the following configuration:

```cpp
constexpr embot::app::theCANboardInfo::applicationInfo applInfo
{
    embot::prot::can::versionOfAPPLICATION {2, 0, 15},
    embot::prot::can::versionOfCANPROTOCOL {2, 0}
};
```

Reply:
Thank you @sgiraz 👍🏻 
@lrapetti could you update the application and protocol versions accordingly?